.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: elixir
+elixir:
+  - 1.4
+addons:
+  postgresql: '9.4'
+env:
+  - MIX_ENV=test
+before_script:
+  - mix do ecto.create
+script:
+  - mix do deps.get, compile --warnings-as-errors, coveralls.json
```

Comment:
we might end up turning _off_ the `--warnings-as-errors` option. given the issues we're having in: https://github.com/dwyl/hits-elixir/issues/19

Reply:
Actually, this should be left on. It does not error for third party warnings, see: https://github.com/dwyl/hits-elixir/issues/19#issuecomment-372829963